rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -689,12 +689,19 @@ stream {
         -- init modules
         local ok, res
 
+        ok, res = pcall(require, "configuration")
+        if not ok then
+          error("require failed: " .. tostring(res))
+        else
+          configuration = res
+          configuration.nameservers = { {{ buildResolversForLua $cfg.Resolver $cfg.DisableIpv6DNS }} }
+        end
+
```

Comment:
Not the concern of this PR, but we should really have just `configuration.lua`. This applies to other stream subsyste Lu code duplication as well.

There's an interface `ngx.config.subsystem` available where we can dynamically check whether it is `http` or `stream` subsystem when the module loads.